app/views/fmeas/edit.html.erb

```diff
@@ -1,5 +1,5 @@
 <%= render 'fmeas/edit_fmea/fmea_header', fmea: @fmea %>
-<div class="container" data-controller="edit-fmea" data-controller-index="">
```

Comment:
I check through the js and couldnt see this being used... so i removed. Let me know if i was mistaken